lib/src/generator/generator.dart

```diff
@@ -20,6 +21,10 @@ String generateFile(Resources res) {
     }
   }
 
+  if (arguments.assetClasses?.isNotEmpty == true) {
+    fullCode += "import 'asset_classes.dart';";
```

Comment:
the imports should be handled by the different generators.
the generators return `DartClass` which have a field called `imports`.
this imports will be aggregated together in the final code